### PR TITLE
Issue 24 show last update time on the web view

### PR DESCRIPTION
**Pull request by caarmen**
* Added a "Last update" column to the web view of the PR list.
* Fixed sorting of the PR list.

### CODE REVIEW

File: gm_pr/PrFetcher.py [hunk 1]
```diff
@@ -78,7 +79,7 @@ def fetch_data(project_name, url, org):
                            is_old=is_old)
             pr_list.append(pr)
 
-    sorted(pr_list, key=lambda pr: pr.updated_at)
+    project['pr_list'] = sorted(pr_list, key=attrgetter('updated_at'), reverse=True)
```
Comment (caarmen) [inline gm_pr/PrFetcher.py]: * Supposedly ```attrgetter``` is faster. (https://wiki.python.org/moin/HowTo/Sorting#Operator_Module_Functions)
* If we don't assign the result of ```sorted``` to the ```project['pr_list']``` field, the result of the sorting is lost.

Comment (CedricCabessa) [inline gm_pr/PrFetcher.py]: > If we don't assign the result of sorted to the project['pr_list'] field, the result of the sorting is lost.

oopsi ;-)

**Comment by CedricCabessa** [comment]
LGTM

**Comment by amasciulli** [comment]
LGTM

File: gm_pr/templates/pr.html [hunk 2]
```diff
@@ -18,6 +20,7 @@
             </tr>
         {% for pr in project.pr_list %}
             <tr class="pr">
+                <td class="last-update">{{pr.updated_at|naturaltime}}</td>
```
Comment (fbarthelery) [inline gm_pr/templates/pr.html]: :+1:

**Comment by fbarthelery** [comment]
LGTM

**Comment by mmaret-geny** [comment]
LGTM

**Pull request merged**